File: PROGRAMMERS/lv0/점의 위치 구하기.js
function solution(dot) {
    return dot[0] > 0 && dot[1] > 0
        ? 1
        : dot[0] < 0 && dot[1] > 0
        ? 2
        : dot[0] < 0 && dot[1] < 0
        ? 3
        : 4;
}

let inps = [
    [2, 4],
    [-7, 9],
];

for (let i of inps) {
    console.log(solution(i));
}

File: PROGRAMMERS/lv0/python/점의 위치 구하기.py
def solution(dot):
    if(dot[0]*dot[1] < 0):
        return 2 if dot[0]<0 else 4
    return 1 if dot[0]>0 else 3

inps = [
    [2, 4],
    [-7, 9]
]

for inp in inps:
    print(solution(inp))